codeship-services.yml

```diff
@@ -40,7 +40,7 @@ e2e:
     - /dev/shm:/dev/shm
 
 db:
-  image: mdillon/postgis:9.6-alpine
+  image: mdillon/postgis:10-alpine
```

Comment:
The migration below was getting a syntax error with the older version of postgres we were previously using. Version 10 is the one we use in prod/feature instances so I went with that 👍